index.html

```diff
@@ -48,7 +48,7 @@ <h2>Chess Battle Royale</h2>
                     style="position:absolute; top: 0; bottom: 0;left: 0;right: 0; margin: auto; bottom: 200px; opacity: 1; visibility: hidden; z-index:1000">
                     <div style="text-align: center; display:block; padding: 5px 10px; background: rgba(0,0,0,0.7); color:white"
                         class="ach_modal_inner">
-                        <h3>Achievement Unlocked!</h3>
+                        <link href="/architement.html">Achievement Unlocked!</link>
```

Comment:
This seems to be a broken link after you removed the architement.html file earlier.